libraries/cyclestreets-view/src/main/java/net/cyclestreets/views/overlay/FindPlaceOverlay.kt

```diff
@@ -0,0 +1,66 @@
+package net.cyclestreets.views.overlay
+
+import android.content.Context
+import android.content.res.Resources
+import android.graphics.Canvas
+import android.graphics.Point
+import android.graphics.Rect
+import android.graphics.drawable.Drawable
+import androidx.core.content.res.ResourcesCompat
+import net.cyclestreets.Undoable
+import net.cyclestreets.view.R
+import net.cyclestreets.views.CycleMapView
+import org.osmdroid.api.IGeoPoint
+import org.osmdroid.api.IProjection
+import org.osmdroid.views.MapView
+import org.osmdroid.views.overlay.Overlay
+
+class FindPlaceOverlay(context: Context, cycleMapView: CycleMapView) : Overlay(), Undoable {
+    // (Passing CMV as parm so centreOn can be accessed, which we need to calc screen coords)
+    private val placeMarker_: Drawable?
+    private var cMapView_: CycleMapView
```

Comment:
My personal view (and one which I have gradually been applying across the codebase) - there's really no need for the trailing underscores in the names of  private vars / vals - I think this is a legacy from when Jez originally wrote the app and Java was younger 😄 

So I would be inclined to remove these from the names.

Reply:
OK, I did wonder about that convention and guessed it was from earlier days, as I see Android Studio now colours them according to scope.  Thanks for confirming that.